ihs/pom.xml

```diff
@@ -34,7 +34,7 @@
 
     <properties>
         <git.proj>azure.websphere-traditional.image</git.proj>
-        <azure.apiVersion>2020-06-01</azure.apiVersion>
-        <azure.apiVersion2>2019-06-01</azure.apiVersion2>
+        <git.repo>WASdev</git.repo>
+        <template.pid.properties.url>${artifactsLocationBase}/${git.proj}/${git.tag}/config.properties</template.pid.properties.url>
```

Comment:
@majguo please add a comment that explains the source of the value for  `git.tag`.  I suspect the source is `-Dgit.tag` at the maven command line, but it needs to be in the comment.

Reply:
@edburns Good catch. Fixed with 1573df4.